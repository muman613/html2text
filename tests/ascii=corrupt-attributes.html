<html>
	<body>
		<font color:="" ?#ffffff?="">blala</font>
		normal text
		<img src:"foo" fno*rds alt="an image" />
	</body>
</html>
