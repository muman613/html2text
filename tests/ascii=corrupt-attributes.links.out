ascii=corrupt-attributes.default.out